contracts/0.8.25/vaults/StakingVault.sol

```diff
@@ -36,6 +36,8 @@ import {ERC1967Utils} from "@openzeppelin/contracts-v5.0.2/proxy/ERC1967/ERC1967
  *   - `withdraw()`
  *   - `requestValidatorExit()`
  *   - `rebalance()`
+ *   - `pauseDeposits()`
```

Comment:
Not in sync with method names

Reply:
fixed in https://github.com/lidofinance/core/pull/922/commits/48bccda5b9e2ac126440877ee587c57f5518c2db